### DISCUSSION TITLE
Tailwind CSS IntelliSense, not working with the theme

### DISCUSSION THREAD

**Discussion by nurielmeni** [tailpress/tailpress#166] [Q&A]
I have tried to install tailwinds in the workspace root directory, with tailwinds.config.js file. But it was still not working.
Any idea for the reason or solution?

**Comment by jeffreyvr** [comment]
Hi @nurielmeni,

I'm assuming you mean the VS Code IntelliSense?

If the workspace root directory isn't the theme itself, you may do the following:

Add a workspace specific `settings.json` file if it doesn't already exist. So: `.vscode/settings.json`

Add the following to the file (replace your-theme-directory-name):

```json
{
    "tailwindCSS.experimental.configFile": "./wp-content/themes/your-theme-directory-name/tailwind.config.js"
}
```
